roofit/hs3/inc/RooFitHS3/RooJSONFactoryWSTool.h

```diff
@@ -140,8 +140,10 @@ class RooJSONFactoryWSTool {
    std::string exportYMLtoString();
    bool importJSONfromString(const std::string &s);
    bool importYMLfromString(const std::string &s);
+   bool importFunctionfromString(const std::string &s);
```

Comment:
```suggestion
   void importFunctionfromString(const std::string &jsonString);
```
You don't even use the return value, right? So this can be a `void` function, which is also consistent with `importFunction()` then.

Also, never use single letters for variable names (I know the source of the `RooJSONFactoryWS` tool is a bad example :)). It makes it difficult to find matching variable names in a file.